src/frontend/src/lib/utils/token.utils.ts

```diff
@@ -141,6 +141,42 @@ export const mapTokenUi = ({
 	})
 });
 
+/** Function to sum the balances of two tokens.
+ *
+ * If the decimals of the tokens are the same, the balances are added together.
+ * If the decimals are different, the function returns null.
+ * If one of the balances is nullish, the function returns the other balance.
+ * If both balances are nullish, the function prioritize the first token
+ * NOTE: the function assumes that the two tokens are always 1:1 twins, for example BTC and ckBTC, or ETH and SepoliaETH
+ *
+ * @param token1
+ * @param token2
+ * @returns The sum of the balances or nullish value.
+ */
+export const sumTokenBalances = ([token1, token2]: TokenUi[]): TokenUi['balance'] =>
```

Comment:
- Should we add a guard if there are more than two tokens?

- If exactly two tokens, should the array be actually typed as `[TokenUi, TokenUi]`?

Reply:
you are right! I was already thinking about multiple tokens in the group! Changed to `[TokenUi, TokenUi]` that should act as guard too, no?

Reply:
Yes, works for me.